### PR TITLE
Actually fix status code

### PR DESCRIPTION
**Pull request by abhinav**

### CODE REVIEW

**Comment by abhinav** [comment]
@junchaowu @blampe

File: tests/schemes/test_thrift.py [hunk 1]
```diff
@@ -1243,3 +1246,45 @@ def testString(request):
...
+    res = yield conn.send(
+        CallRequestMessage(
+            service=b'service',
+            headers={b'cn': b'client', b'as': b'thrift'},
+            args=[
+                b'ThriftTest::testException',
+                b'',
+                bytearray([
+                    0x0B,        # type = string
+                    0x00, 0x01,  # field ID 1
```
Comment (blampe) [inline tests/schemes/test_thrift.py]: nasty

**Comment by blampe** [comment]
ok

**Comment by junchaowu** [comment]
lgtm

**Pull request merged**